dist/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+  <title>Document</title>
+<script defer src="bundle.js"></script></head>
+<body>
+  <div class="app-container">
+    <div class="title">
+      <h1>To Do List</h1>
+      <i class="bi bi-arrow-repeat pointer"></i>
+    </div>
+    <div class="list">
+      <div class="add-item">
+        <ul>
+          <li>
+            <form class="form">
+              <input type="text" placeholder="Add to your list...">
+              <i class="bi bi-arrow-90deg-down pointer"></i>
+            </form>
+          </li>
+        </ul>
+      </div>
+      <div class="list-items">
+        <ul id="list-container">
+          <!-- <li class="item-container">
+            <form>
+              <input type="checkbox" name="item-1" id="item-1">
+              <label for="item-1">Thing to do!</label>
+            </form>
+            <i class="bi bi-three-dots-vertical"></i>
+          </li> -->
```

Comment:
- [ ] Please avoid leaving commented code to improve the quality of your code and make it clean